package.json

```diff
@@ -30,30 +30,28 @@
     "lerna": "^6.0.0"
   },
   "devDependencies": {
-    "@inrupt/eslint-config-base": "0.4.0",
-    "@inrupt/eslint-config-lib": "0.4.1",
+    "@inrupt/eslint-config-base": "^1.3.0",
+    "@inrupt/eslint-config-lib": "^1.3.0",
+    "@inrupt/jest-jsdom-polyfills": "^1.3.0",
+    "@peculiar/webcrypto": "^1.4.0",
```

Comment:
This should be done in `@inrupt/jest-jsdom-polyfills`

Reply:
Waiting on https://github.com/inrupt/javascript-style-configs/pull/62

Reply:
Approved

Reply:
Now for the release :) https://github.com/inrupt/javascript-style-configs/pull/63